openslides_backend/main.py

```diff
@@ -45,13 +47,28 @@ def __init__(self, view_name: str, *args: Any, **kwargs: Any) -> None:
 
         super().__init__(*args, **kwargs)
 
+    def loglevel(self) -> str:
+        if is_dev_mode():
+            return "DEBUG"
```

Comment:
I think if given, the explicit log level should override the default one from the dev mode (and I think @ostcar also wanted this to be able to remove the log output while testing in dev mode). So, the handling should be the following:
- `OPENSLIDES_LOGLEVEL` set? use the given level
- else:
  - dev mode? -> `DEBUG`
  - else -> default log level `INFO`

Also for unity purposes, we should then put all default values into variables (like you already started doing) and use variables for the names of env vars (like in `env.py`)

Reply:
Refactored